pkgs/clickable-apis/index.ts

```diff
@@ -13,11 +13,69 @@ import {
   createNodePayloadClient,
 } from 'simple-payload'
 
+/**
+ * Interface for the API header object that appears at the top of all JSON responses
+ */
+export interface ApiHeader {
+  /** Name of the API */
+  name: string
+  /** Description of the API */
+  description: string
+  /** Home URL */
+  home: string
+  /** Login URL */
+  login: string
```

Comment:
login and signup should be optional, because they will not exist when logged in ... just like upgrade won't exist when not logged in, or when on the max enterprise plan that can't be upgraded

also, we shouldn't ever use snake_case, we only use camelCase, but `github_issues` is too verbose and should just be issues